scripts/jenkins/cloud/ansible/roles/list_packages/tasks/list-packages.yml

```diff
@@ -0,0 +1,41 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Collect list of installed packages in plaintext
+  shell: |
+    rpm -qa | sort > /tmp/rpms/rpms_{{ inventory_hostname }}_{{ suffix }}.txt
```

Comment:
cant you just use 
```
- name: Gather the packages
  package_facts:
```

here? It will also get you an ansible friendly output that you can save in any format using `to_yaml` or `to_json`

Reply:
Hmmm, sounds interesting. I ll test that.

Reply:
In my case unfortunately it complicate things. There is a package prereq. `rpm-python` which is not present in our images and that would bring more additional steps like add repos and remove after